common/option-model.cpp

```diff
@@ -38,6 +38,7 @@ namespace rs2
             {
                 option.range = options->get_option_range(opt);
                 option.read_only = options->is_option_read_only(opt);
+                option._volatile = options->is_option_volatile( opt );
```

Comment:
Why the `_` when all other fields doen't have it? looks weird

Reply:
Got it :)
If we make additional changes please consider adding a comment `adding _ prefix due to volatile is a saved keyword`
at the parameter definition?

Reply:
Sure.
If you prefer I can make a commit that adds the prefix to all members :-)